### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by harjotgill**

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## 1.1.1 - 03/20/2017
+
+* Topology Maps
+    * Multiple topology maps -- Checkout the new topology maps page to see bundled maps and maps created by other users. You can also use the topology sandbox to explore and create different logical and physical topologies maps.
```
Comment (jarnoux) [inline docs/release-notes.md]: there will be no "bundled" maps. Maybe also call them "precanned" to be consistent with the way we call dashboards and alerts

Comment (jarnoux) [inline docs/release-notes.md]: will there be bundled maps?

Comment (harjotgill) [inline docs/release-notes.md]: Fixed.

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## 1.1.1 - 03/20/2017
+
+* Topology Maps
+    * Multiple topology maps -- Checkout the new topology maps page to see pre-canned maps and maps created by other users. You can also use the topology sandbox to explore and create different logical and physical topologies maps.
```
Comment (tanveergill) [inline docs/release-notes.md]: typo: `topologies maps` -> `topology maps`

**Pull request merged**